presentation/src/main/java/me/androidbox/presentation/event/viewmodel/EventViewModel.kt

```diff
@@ -63,6 +63,27 @@ class EventViewModel @Inject constructor(
                     )
                 }
             }
+            is EventScreenEvent.OnSaveEventDetails -> {
+                /* TODO Get all the information related from teh EventScreenState
+                *       and create the Event object */
+
+                /* Then insert this event into the room db
+                val Event = Event(
+                    id = UUID.randomUUID().toString(),
+                    description = eventScreenState.value.saveEditOrDescriptionContent,
+                    photos = eventScreenState.value.listOfPhotoUri,
+                    isUserEventCreator = true,
+                    ...
+                )
+                   insertEventDetails(event)
+                 */
```

Comment:
So it will be something like the above.
1) Extract the information from the EventScreenState i.e. title, description, photo urls, id, etc.
2) Create the Event object
3) Insert that event into the DB

Reply:
Looks good